game/static/game/css/board.css

```
@@ -600,6 +600,52 @@ body {
+}
+
+.countdown-number {
+    font-family: 'Cinzel', serif;
+    font-size: clamp(3rem, 12vw, 6rem);
+    font-weight: 700;
+    color: #f0c040;
+    text-shadow: 0 0 20px rgba(240, 192, 64, 0.6);
+    animation: countdownPop 1s ease-in-out;
+}
+
+@keyframes countdownPop {
```

Stylelint (17.14.0)

[error] Expected keyframe name "countdownPop" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)